TLSharp.Tests.VS/TLSharpTestsVs.cs

```diff
@@ -0,0 +1,71 @@
+﻿
+using System.Threading.Tasks;
+
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace TLSharp.Tests
+{
+    [TestClass]
+    public class TLSharpTestsVS : TLSharpTests
```

Comment:
Exactly, the approach that @aarani describes is the one used in https://github.com/sochix/TLSharp/pull/289 . You guys have the choice of what to merge. I could also create a v3 that just gets rid of the MSTest infrastructure altogether, as @sochix suggested in the other pull request. Please let me know how to proceed. If NUnit support doesn't get merge, IMHO it makes opensource developers like me not really comfortable to work in the library because we depend on a Windows VM running Visual Studio :(